src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -56,21 +56,45 @@ class IfTypes(int, Enum):
 class ArpUpdater(MIBUpdater):
     def __init__(self):
         super().__init__()
+        self.db_conn = Namespace.init_namespace_dbs()
         self.arp_dest_map = {}
         self.arp_dest_list = []
         self.arp_dest_map = {}
         self.arp_dest_list = []
+        self.neigh_key_list = {}
 
-    def update_data(self):
-        self.arp_dest_map = {}
-        self.arp_dest_list = []
+    def reinit_data(self):
+        if len(self.db_conn) > 1:
+            Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB)
+            self.neigh_key_list = Namespace.dbs_keys_namespace(self.db_conn, mibs.APPL_DB, "NEIGH_TABLE:*")
```

Comment:
>dbs_keys_namespace [](start = 44, length = 18)

Function does not exist.

Reply:
Requires PR#137

Reply:
Added function as PR #137 is not merged yet.